File: Langauge/Java/Reactive/02_Reactor/Tutorial/app/src/main/java/tutorial/dto/TestDTO.java
package tutorial.dto;

public class TestDTO {
    private String name;
    private Integer age;

    public TestDTO() {
    }

    public TestDTO(String name, Integer age) {
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public Integer getAge() {
        return age;
    }

    public void clear() {
        this.name = null;
        this.age = null;
    }
}

File: Langauge/Java/Framework/mybatis/basic/src/main/kotlin/com/example/mybatis/dto/TestDTO.kt
package com.example.mybatis.dto

import java.time.LocalDateTime


data class TestDTO(

    var id: Long? = null,

    var name: String? = null,

    var age: Int? = null,

    var email: String? = null,

    var createAt: LocalDateTime? = LocalDateTime.now(),

    var updateAt: LocalDateTime? = null,

    var deleteAt: LocalDateTime? = null,

)
